### DISCUSSION TITLE
How do I allow --upload-nsfw with either cli commands or any other way

### DISCUSSION THREAD

**Discussion by Rinkakazi** [tteck/Proxmox#3413] [Help Needed]
I would like to allow nsfw images as i keep getting the same error but after having changed the config file and rebooting it had no effect and after trying to use the cli commands by typing:

`set -o allexport; source /opt/photoprism/config/.env; set +o allexport`
`cd /opt/photoprism/bin/`
`./photoprism start --upload-nsfw`

This had no effect and after trying to use chatgpt to understand the output of the photoprism help command i did not succeed

PS: im a linux noob

**Comment by ChristophCaina** [comment]
hi there,
any success?

I'm having the same issue ...
